source/tutorials/datapacks.md

```diff
@@ -0,0 +1,338 @@
+| Type      | The supported format for a specific context. Includes Objects, Arrays, Strings, Booleans, Integers, and Doubles. |
+| Object    | In the context of the JSON standard, a group of key-value pairs held together by a pair of curly brackets (<code>{ }</code>). All JSON files must be a JSON Object, and name-value pairs may use an Object as a value. |
+| Array     | In the context of the JSON standard, a group of value types held together by a pair of square brackets(<code>[ ]</code>). JSON Arrays may contain multiple Values, or multiple Objects, but not name-value pairs directly. |
+| String    | A set of characters. In JSON, strings must always be within quotation marks (<code>" "<code>). |
+| Boolean   | The values <code>true</code> or <code>false</code>, not contained within quotation marks. |
+| Integer   | A whole number. For this document, between positive and negative two billion, not contained within quotation marks. You generally won't use numbers that high. |
+| Double    | A number, including decimal numbers, not contained within quotation marks. |
+| Name-Value Pair | In the context of the JSON standard, a string key and an matching value, usually in the format <code>"name": value</code>. Name-value pairs that are not the last name-value pair in an Object must be separated by a comma (<code>,</code>). |
+| Name      | In the context of the JSON standard, the left half of a name-value pair. Must be a String. Only one occurance of a name will be read in a single object's top level, usually the first, though sibling in an array or descendants in objects may hold the same Name. |
+| Value     | In the context of the JSON standard, the right half of a name-value pair. May be any type that matches the context. Values within quotation marks (<code>" "</code>) are treated as Strings. |
+| Translation Key | A specially formatted string, which will attempt to be processed through the Minecraft translation file. If the client language file contains a matching Name, substitutes the corresponding Value, otherwise, presents the key to the user directly. |
+
+### Example Folder Layout
+
+A complex data pack can have many files across many types in many namespaces, as shown below. Smaller data packs consisting of tweaks or settings changes may only have two or three files.
+
+<img src="../../assets/images/misc/datapack_example.png" class="img-fluid mx-auto" alt="Example Complex Data Pack Layout">
+
+<p style="font-size:12pt;text-align:center"><b>The pack.mcmeta file is mandatory, and Minecraft will not load a data pack without one, or with an improperly formatted one.</b></p>
+
+For Minecraft 1.16, a typical pack.mcmeta file looks like this: 
+
+<pre><code>{
+  "pack": {
+    "pack_format": 6,
+    "description": "Rename To Your Preferences"
+  }
+}
+</code></pre>
+
+The <code>"description"</code>'s value is displayed to the user, so it's best to make it descriptive and unique.
```

Comment:
in all of my experiences with these, they end up being more of titles than descriptions; maybe note that here